administration/src/util/translations/de.json

```diff
@@ -65,9 +75,62 @@
+    "regionAlertPostalCode": "Bitte geben Sie oben eine 5-stellige Postleitzahl an, sodass die zuständige Behörde automatisch ermittelt werden kann.",
+    "regionNotDetermined": "Leider konnte die zuständige Behörde nicht automatisch anhand Ihrer Postleitzahl ermittelt werden. <br />Bitte nutzen Sie das folgende Auswahlfeld, um Ihre zuständige Behörde auszuwählen.",
+    "regionNotUnique": "Ihr Postleitzahlen-Gebiet ist nicht eindeutig einer Region zuordenbar. Bitte wählen Sie die korrekte Region<br />Ihres Hauptwohnsitzes aus der Liste:",
+    "regionDetermined": "Die zuständige Behörde konnte anhand Ihrer Postleitzahl automatisch ermittelt werden.",
+    "regionSelectionDescriptionStart": "In der Regel müssen Sie die Ehrenamtskarte in dem Landkreis oder der kreisfreien Stadt Ihres Hauptwohnsitzes beantragen. Weitere Informationen können Sie ",
+    "regionSelectionDescriptionEnd": "einsehen",
+    "regionSelectionListTextStart": "Eine Liste der teilnehmenden Landkreise und kreisfreien Städte für den zentralen Beantragungsprozess finden Sie im Auswahlfeld unten.<br />Alle anderen Regionen können Sie",
+    "regionSelectionListTextEnd": "direkt kontaktieren, um Ihre Ehrenamtskarte zu erhalten.",
+    "regionSelectionLabel": "Empfänger des Antrags",
+    "personalAddress": "Adresse (Erstwohnsitz)",
+    "personalFurtherInformation": "Weitere Angaben",
+    "organizationContactPersonDescription": "Bitte geben Sie hier die Daten der Person an, die ihr ehrenamtliches Engagement bestätigen kann.",
+    "organizationContactPersonAlert": "Die Kontaktperson der Organisation und die antragsstellende Person scheinen identisch zu sein. <br/>Bitte beachten Sie, dass Anträge auf dieser Grundlage nicht bewilligt werden können.",
+    "juleicaCardAttachmentTitle": "Kopie der JuLeiCa",
+    "juleicaCardAttachmentDescription": "Hängen Sie hier bitte Ihre eingescannte oder abfotografierte JuLeiCa an.",
+    "honoredByMinisterDivider": "Angaben zum Ehrenzeichen",
+    "honoredByMinisterCertificate": "Urkunde",
+    "honoredByMinisterCertificateDescription": "Hängen Sie hier bitte Ihre eingescannte oder abfotografierte Urkunde an.",
+    "goldenCardRequirementsTitle": "Ich erfülle folgende Voraussetzung für die Beantragung einer goldenen Ehrenamtskarte:",
+    "blueCardRequirementsTitle": "Ich erfülle folgende Voraussetzung für die Beantragung einer blauen Ehrenamtskarte:",
+    "applicationStepPersonalData": "Persönliche Angaben",
+    "applicationStepRequirements": "Voraussetzungen",
+    "applicationStepSubmit": "Antrag Senden"
   },
   "applications": {
+    "_comment": "!Important! These keys should not be edited, since they are stored in the database and have to remain resolvable. If you change these keys, labels of old applications can not be resolved anymore.",
```

Comment:
it should be under "application" I guess, not "applications"

Reply:
Ich could be applicationOverview maybe

Reply:
hm i checked the content of the namespaces and overview is not fitting. seems like its kind of mixture unfortunately `applicationCard` could be good

Reply:
https://github.com/digitalfabrik/entitlementcard/issues/1887